code/addons/essentials/src/index.ts

```diff
@@ -88,9 +88,9 @@ export function addons(options: PresetOptions) {
 
   // NOTE: The order of these addons is important.
   return [
-    'docs',
     'controls',
     'actions',
+    'docs',
```

Comment:
This will be applied to all the renderers, is there any reason why you changed the order? it is important

Reply:
I just wanted to show the Code panel after controls and actions so it is not the first panel.
Is there any other way to achieve this?